src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer
+    .filter(({ title, description }) => {
+      return (
+        title.toLowerCase().includes(query.toLocaleLowerCase())
+        || description.toLowerCase().includes(query.toLocaleLowerCase())
```

Comment:
You can concatenate a title and description to simplify your code. Also, you probably don't want to filter an array if there is no query.